renn/rnn/network.py

```diff
@@ -135,3 +135,27 @@ def eigsorted(jac):
 def timescale(eigenvalues):
   """Converts eigenvalues into approximate time constants."""
   return -1. / np.log(np.abs(eigenvalues))
+
+def positional_encoding_matrix(seq_len, d_model, timescale=10000):
```

Comment:
Let's just call it `positional_encoding`? matrix adds unnecessary length

Reply:
and this is less important, but maybe `dim` or `model_dim` for the dimension?

Reply:
I did `embed_dim` since `model_dim` might get confused with the actual number of units.